src/resources/js/tickets.js

```diff
@@ -989,6 +989,35 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 			showButtonPanel: false,
 			onChange       : function() {
 			},
+			beforeShow     : function( element, object ) {
+				object.input.data( 'prevDate', object.input.datepicker( 'getDate' ) );
+
+				// Capture the datepicker div here; it's dynamically generated so best to grab here instead of elsewhere.
+				var $dpDiv = $( object.dpDiv );
+
+				// "Namespace" our CSS a bit so that our custom jquery-ui-datepicker styles don't interfere with other plugins'/themes'.
+				$dpDiv.addClass( 'tribe-ui-datepicker' );
+
+				// Look into making this also compatible with ACF
+				// $event_details.trigger( 'tribe.ui-datepicker-div-beforeshow', [ object ] );
+
+				$dpDiv.attrchange({
+					trackValues : true,
+					callback    : function( attr ) {
+						// This is a non-ideal, but very reliable way to look for the closing of the ui-datepicker box,
+						// since onClose method is often occluded by other plugins, including Events Calender PRO.
+						if (
+							attr.newValue.indexOf( 'display: none' ) >= 0 ||
+							attr.newValue.indexOf( 'display:none' ) >= 0
+						) {
+							$dpDiv.removeClass( 'tribe-ui-datepicker' );
+
+							// Look into making this also compatible with ACF
+							// $event_details.trigger( 'tribe.ui-datepicker-div-closed', [ object ] );
```

Comment:
Should this be removed or uncommented?

Reply:
I wanted to leave it there since I just ported it from TEC and wanted to move it into a Ticket for the MR to add the ACF compatibility.